apibindings/go/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/google/osv.dev/apibindings/go
```

Comment:
Do we want to set up the `osv.dev` vanity URL? And thoughts on this just being `bindings` instead of `apibindings`?

```golang
import "osv.dev/bindings/go/osvdev"
```

Reply:
I guess I can imagine other bindings? e.g. instead of a python library we might have a go library as well, but I think that's thinking too far ahead and you're right we should just stick to bindings.

Reply:
Sure we can do a vanity URL, I'll make a separate PR for it.